admission-policy/readme.adoc

```diff
@@ -0,0 +1,342 @@
+
+= Admission Control for Kubernetes on AWS
+Doc Writer <tim@styra.com>
+v1.0, 2018-01-17
+
+Kubernetes https://kubernetes.io/docs/admin/admission-controllers/[Admission Controllers]
+perform *semantic validation* of resources during create, update, and delete operations.
+In Kubernetes 1.7+, you can use OPA to enforce custom policies without recompiling or reconfiguring
+the Kubernetes API server by leveraging
+https://kubernetes.io/docs/admin/extensible-admission-controllers/#external-admission-webhooks[External Admission Webhooks].
+
+For example, you could enforce any of the following policies:
+
+* All images come from an AWS repository (other than a whitelist)
+* Images are pulled from the same AWS repository in the same region as the cluster
+* Namespaces are assigned to different AWS IAM groups
+
+
+== Goals
+
+This tutorial shows how to use OPA to enforce custom policies on resources in
+Kubernetes running on AWS. For the purpose of this tutorial, you will define a policy that
+prevents users from running `kubectl exec` on "privileged" containers in the
+"production" namespace.
```

Comment:
"production", 'production' and `production` are three different usages of the word production. This need to be consistent.

Reply:
Fixed